doc/guides/test/sanitycheck.rst

```diff
@@ -49,12 +50,13 @@ all possible tests using the following options:
         $ ./scripts/sanitycheck --all --enable-slow
 
 This will build for all available boards and run all applicable tests in
-a simulated (QEMU) environment.
+a simulated (for example QEMU) environment.
 
-The sanitycheck script accepts the following optional arguments:
+The sanitycheck script accepts the following optional arguments::
```

Comment:
This looks amazingly like (some of) the output from ``sanitycheck --help`` so should we be pulling this into the documentation directly from the docstring vs. copying it?  Also the display (as a code block) is rather hard to read without a blank lines between arguments.

Reply:
I would prefer this if possible, they are likely to fall out of sync otherwise

Reply:
We can just skip altogether, I was just updating it and formatting did not work out so I put it in a code bock.